<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <meta property="og:url" content="http://www.hu-jun.com" />
  <meta property="og:type" content="website" />
  <meta property="og:title" content="Jun Hu's Portfolio" />
  <meta property="og:description" content="Programmer" />
  <meta property="og:image" content="http://www.your-domain.com/path/image.jpg" />

  <link rel="icon" type="image/png" href="../img/sunshine.png" izes="256x256" />
  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.4.1/css/all.css" integrity="sha384-5sAR7xN1Nv6T6+dT2mhtzEpVJvfS3NScPQTrOxhwjIuvcA67KV2R5Jz6kr4abQsz"
    crossorigin="anonymous">
  <link rel="stylesheet" type="text/css" href="css/academicons-1.8.0/css/academicons.css">
  <link rel="stylesheet" href="../css/main.css">
  <title>Large-scale Sequencing</title>
</head>

<body>
  <header>
    <div class="menu-btn">
      <div class="btn-line"></div>
      <div class="btn-line"></div>
      <div class="btn-line"></div>
    </div>

    <nav class="menu">
      <div class="menu-branding">
        <div class="portrait"></div>
      </div>
      <ul class="menu-nav">
        <li class="nav-item">
          <a href="../index.html" class="nav-link">
            Home
          </a>
        </li>
        <li class="nav-item">
          <a href="../about.html" class="nav-link">
            About
          </a>
        </li>
        <li class="nav-item current">
          <a href="../projects.html" class="nav-link">
            Projects
          </a>
        </li>
        <li class="nav-item">
          <a href="../contact.html" class="nav-link">
            Contacts
          </a>
        </li>
      </ul>
    </nav>
  </header>

  <main id="prj">
    <h1 class="project-name">
      Large-scale Sequencing
    </h1>
    <div class="prj-block">
      <a href="../projects.html" class="button button-green">Back To Projects</a>
    </div>

    <div class="prj-block">
      <a href="https://github.com/vibrioh/seq" target="_blank" class="button button-red">GitHub</a>
    </div>

    <div class="prj-block">
      <a href="https://www.ncbi.nlm.nih.gov/pubmed/29735531" target="_blank" class="button button-blue">Research
        Paper
        Published On Top Journal</a>
    </div>





    <h2 class="sm-heading">

    </h2>

    <div class="project-area">
      <h2 class="section">Introduction</h2>
      <p>In this study, we performed tissue specific deletion to determine the role of Smad4 in chondrocyte
        hypertrophic differentiation during endochondral bone formation in mouse limb. We found that loss of Smad4
        within the limb resulted in absence of chondrocyte hypertrophy and ossification in humerus. To further
        determine how the skeletal defects arise in the mutants, we performed RNA-Seq along with ChIP-Seq analysis and
        found that Smad4 directly binds to Runx2 regulatory elements. Our data suggest a mechanism that Smad4 controls
        chondrocyte hypertrophy through regulating Runx2 expression during mouse skeletal development. The regulatory
        mechanism between Smad4 and Runx2 provides critical insights into the pathogenesis of chondrodysplasia.</p>
      <h2 class="section">Implementation</h2>
      <p>RNA-seq analysis in the next step involves alignment. Bowtie (http://bowtie-bio.sourceforge.net/index.shtml),
        Burrows-Wheeler Aligner (BWA http://bio-bwa.sourceforge.net/), Maq (http://maq.sourceforge.net/),
        Mosaik(http://gkno.me/), NovoAlign (http://www.novocraft.com/products/novoalign/), SEAL
        (http://biodoop-seal.sourceforge.net/), Stampy (http://www.well.ox.ac.uk/project-stampy) and so on are some
        short, unspliced aligners. STAR (http://code.google.com/p/rna-star/), Bowtie2
        (http://bowtie-bio.sourceforge.net/bowtie2/index.shtml), TopHat
        (https://ccb.jhu.edu/software/tophat/index.shtml) and so on are examples of spliced/gapped aligners.
        Normalization, quantitative analysis and differential expression analysis can be done using BaySeq
        (http://bioconductor.org/packages/release/bioc/html/baySeq.html), Cufflinks/Cuffdiff
        (http://cole-trapnell-lab.github.io/cufflinks/),
        DESeq(http://bioconductor.org/packages/release/bioc/html/DESeq.html),
        EdgeR(https://bioconductor.org/packages/release/bioc/html/edgeR.html),
        NOISeq(https://www.bioconductor.org/packages/release/bioc/html/NOISeq.html), HTSeq
        (http://www-huber.embl.de/HTSeq/doc/install.html) and so on. MACS, Model-based Analysis for ChIP-seq allows a
        more sensitive and robust prediction of peaks using dynamic Poisson distribution. It models the length of the
        sequenced ChIP fragments to improve the spatial resolution of predicted binding sites
        (http://liulab.dfci.harvard.edu/MACS/).</p>
      <h2 class="section">Highlight</h2>
      <p>
        Experienced with Minerva supercomputer. <br>
        Linux system operation. <br>
        Huge data retrieval and process. <br>
        Lots of scientific computing packages. <br>
        R ggplot2 visualization. <br>
        Venn diagram. <br>
        Histogram.
      </p>
      <h2 class="section">Achievement</h2>
      <p>
        This was an entirely unexpected project. As a cardiology-related lab, we were trying to find cardiac defects by
        conditional knocking out of SMAD4, but we did not found anything wrong in the heart. However, we found loss of
        bones in the limbs. So this project is now cooperating with a bone-related research lab for further
        discoveries. (Updated: the research was published on The Journal of Biological Chemistry in 2018)
      </p>



    </div>


  </main>

  <footer id="main-footer">
    Copyright &copy; 2018
  </footer>

  <script src="../js/main.js"></script>
</body>

</html>